deploy/sample/lvmsnapclass.yaml

```diff
@@ -0,0 +1,8 @@
+kind: VolumeSnapshotClass
```

Comment:
There is already an [lvmsnapshot](https://github.com/openebs/lvm-localpv/blob/master/deploy/sample/lvmsnapshot.yaml) file. It can be split and used.